sdk/ml/azure-ai-ml/azure/ai/ml/_schema/job/job_limits.py

```diff
@@ -39,5 +39,7 @@ def make(self, data, **kwargs):
 
 class DoWhileLimitsSchema(metaclass=PatchedSchemaMeta):
     max_iteration_count = fields.Int(
-        metadata={"description": "The max iteration for do_while loop."}, validate=validate.Range(min=1, max=1000)
+        metadata={"description": "The max iteration for do_while loop."},
+        validate=validate.Range(min=1, max=1000),
```

Comment:
@D-W- pls remind me what's our previous discussion result on this

Reply:
I have summarized related discussion in my work item, which shall help